code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -45,6 +45,11 @@
 					movement_target = null
 					stop_automated_movement = 0
 					break
+			for(var/obj/item/toy/cattoy/T in view(1,src))
+				if (!T.cooldown)
+					emote("me", 1, "bats \the [T] around with its paw!")

```

Comment:
instead of its, try \her, it'll correct to the cat's gender.
eg: "with his paw", "with her paw"